src/panels/config/blueprint/ha-blueprint-overview.ts

```diff
@@ -242,9 +242,9 @@ class HaBlueprintOverview extends LitElement {
     });
   }
 
```

Comment:
Let's make a second function that handles the click on the fab:
```ts
private _addBlueprintClicked() {
	this._addBlueprint();
}
```

Reply:
oh man.... I did that first... then thought, nah, Bram will not like that...